.rubocop.yml

```diff
@@ -0,0 +1,32 @@
+AllCops:
+  DisplayCopNames: true
+  Exclude:
+    - example/**/*
```

Comment:
I do locally :-)

Will see if there's a way I can move that exclusion out of this file